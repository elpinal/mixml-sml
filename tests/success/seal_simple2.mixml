link X =
  link X = {module t = [type :Type]} with
  {module v = [val :ext X.t]}
seals
link X = {module t = [type int]} with
{module v = [val 33]}
