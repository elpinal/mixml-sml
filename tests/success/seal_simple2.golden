{t : [= !0], v : [!0]+}
